w3af/core/ui/api/main.py

```diff
@@ -24,17 +24,13 @@
 
```

Comment:
We don't need the exec permissions for this file `100644 → 100755`, please move back to 644